contracts/HealthPass721.sol

```diff
@@ -11,24 +11,59 @@ contract HealthPass721 is ERC721URIStorage, Ownable {
     using Counters for Counters.Counter;
     Counters.Counter private _tokenIds;
 
+ //We can brainstorm about it
+    struct VaccineDetails {
+     string category;
```

Comment:
Not a review comment, but curious about what category is?

Reply:
its just an attribute, something like the main dose or booster dose, or more types